Gemfile

```diff
@@ -11,38 +13,51 @@ def location_for(place, fake_version = nil)
 end
 
 group :system_tests do
-  gem 'rake'
-  gem 'beaker-rspec', *location_for(ENV['BEAKER_RSPEC_VERSION'] || '~> 6.0')
-  gem 'beaker-pe', *location_for(ENV['BEAKER_PE_VERSION'] || '~> 1.1')
-  gem 'listen', '<3.1.0'
+  gem 'rake',                    '~> 0'
+  gem 'beaker-rspec',            *location_for(ENV['BEAKER_RSPEC_VERSION'] || '<  5.7') if RUBY_VERSION <  '2.1.5'
```

Comment:
oh, you are right of course.  yeah i've used the above and `between` before.  it "works" here so i totally forgot.  will fix